lib/capybara/minitest.rb

```diff
@@ -2,10 +2,17 @@
 
 require 'minitest'
 require 'capybara/dsl'
+require 'capybara/scoping'
 
 module Capybara
   module Minitest
     module Assertions
+      include Capybara::Scoping
+
+      def root_scope
+        page
+      end
```

Comment:
@twalpole does this feel like an appropriate abstraction to introduce to share this behavior?